spec/acceptance/new_module_spec.rb

```diff
@@ -27,5 +27,15 @@
                                                                     'operatingsystemrelease' => ['8']))
       end
     end
+
+    describe file('foo/README.md') do
```

Comment:
These new examples are dependent on the module template providing these files. I think it's more valuable to have these examples, with the risk of pdk-module-template breaking them; than to not have them 👍